test/cmdlineTests/standard_import_evmasm_invalid_input/input.json

```
@@ -0,0 +1,33 @@
+{
+    "language": "EVMAssembly",
+    "sources": {
+        "A": {
+            "invalidAssemblyJsonObject": {
+                ".code": [
+                    {
+                        "begin": 36,
+                        "end": 51,
+                        "name": "PUSH",
+                        "source": 0,
+                        "value": "0"
+                    }
+                ],
+                "sourceList": [
+                    "<stdin>"
+                ]
+            }
+        }
+    },
+    "settings": {
+        "outputSelection": {
+            "*": {
+                "": [
+                    "evm.assembly",
+                    "evm.legacyAssembly",
+                    "evm.bytecode",
+                    "evm.deployedBytecode"
+                ]
```

Actually, can't you just use the wildcard here?
```suggestion
                "": ["*"]
```

If not, I'd request only `bytecode` in tests for invalid cases (where we don't get any output anyway) to reduce test verbosity a bit more.